lib/questions.rb

```diff
@@ -1,29 +1,32 @@
 class Questions
-  attr_reader :player_counter
 
-  def initialize()
-    @player_counter = 2
-  end
-
-  def increase_player_counter
-    @player_counter += 1
-  end
+  def turn_count(board)
```

Comment:
Your questions class should not be responsible for figuring out the turn_count.

Reply:
ahhh gotcha, I understand what you meant now 👍🏽